lib/share.js

```diff
@@ -68,8 +68,11 @@ module.exports = function sync (type, opts, dat) {
       // File Imports
       progressOutput[2] = 'Importing new & updated files to archive...'
 
-      // TODO: allow live: true
-      importStatus = dat.importFiles({live: false, resume: true}, function (err) {
+      importStatus = dat.importFiles({
+        live: false, // TODO: allow live: true. opts.live is used by archive.live though =(
```

Comment:
Yeah that would be a great change. (can keep `live` as backwards compat but use `watch` in the docs and in this repo)

So watching files still doesn't work well here with sync?